### ISSUE TITLE
Second beta 0.9.8 released

### ISSUE THREAD

**Issue by rsjaffe** [rsjaffe/MIDI2LR#199]
https://github.com/rsjaffe/MIDI2LR/releases/tag/v0.9.8

Same numbering as first beta release. This fixes the contrast issue and limits dialog error. It also redistributes the commands (there are so many now!). All are listed in the wiki under the [Commands page](https://github.com/rsjaffe/MIDI2LR/wiki/Commands).

**Comment by StevenKersting** [comment]
I was still getting the "bad argument #1to 'find' " error on loading (several times) as well as the limits dialog error (one time) initially. Cleared/resaved preferences/options and it seems ok now (several loads/restarts).

No matter how I go about switching versions it aways says it's 9.8 in the app? (I've done everything to include removing it, shutting down LR, and rebooting the computer before loading it again).

And I am the temp adjustment is still non-functional (regardless of version?). I'm thinking an uninstall/reinstall of LR might be necessary?
<img width="651" alt="screen shot 2016-02-24 at 3 38 26 pm" src="https://cloud.githubusercontent.com/assets/16807274/13300347/aa4b7754-db0d-11e5-9ba3-f75cd9af077b.png">

**Comment by rsjaffe** [comment]
Ok. Couple of observations. It's best not to put the plugin in the program directory. That causes all sorts of problems due to write permissions. See the FAQ for more info. Two, you've somehow mixed several versions of the plugin and app together. Delete everything from your plugin directory, and unzip the latest version, including MIDI2LR.exe, into the plugin directory. Then your versions should be coordinated. I wouldn't uninstall reinstall LR.

**Comment by StevenKersting** [comment]
It shouldn't be permissions. But I moved the plugin anyway. It seems the issue is with OSX's security and the plugin being unsigned. To allow the plugin I have to either allow it individually, or change the security settings globally. Either way, the security only allows/loads the last "approved" version to load (it seems to direct by name, and not location/file info). 
The solution I found was to only have one version on the machine and then approve that one to run. A simple folder renaming/etc doesn't work. A complete directory change might (not sure), but deleting other versions (and restoring if needed) is just as easy.

After all of that I can load/sync the plugin correctly w/o issue/confusion. But, the temp adjustment is still not working (regardless of version). Since it was working in previous versions, and it now isn't working in previous versions, I suspect the issue is local. The app does see the controller adjustment and has it assigned to the right adjustment...the slider just doesn't move. 

I'll keep looking for an answer...

**Comment by StevenKersting** [comment]
Ok, I think I have found the issue. After reloading everything and setting the options from scratch the "other" page was blank... after saving the options I started getting the "bad argument" internal error.

I believe the missing limits (other) page is probably what is causing both the internal error and the temp input failure.
<img width="400" alt="screen shot 2016-02-23 at 4 01 35 pm" src="https://cloud.githubusercontent.com/assets/16807274/13328739/6022e7e6-dbbe-11e5-8825-f1aa7c1d029c.png">

**Comment by StevenKersting** [comment]
Fixed.
I renamed the LR preferences file and attempted to load the options. It failed with the warning above. LR then wrote a new preferences file. I deleted the new file and renamed the old one back again. 

For whatever reason, this restored the options and the "other" panel is now available. And as I suspected, the temp adjustment limits were matched in the middle (i.e. not "default" and no offset/adjustability). I reset it to default and the temp slider is now working again.

I'll reload/restart everything several times to verify stability/consistency. I have no idea "where" this bad information was being stored as I had deleted/reloaded/restarted everything, to include LR itself. I suspect it's got something to do with mac's SMC/NVRAM.
